src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -260,7 +260,7 @@ Initialization.registerNamedMethod('logCustomEvent', (element: HTMLElement, cust
  * A `Search` event is actually sent to the Coveo Usage Analytics service only after the query successfully returns (not
  * immediately after calling this method). Therefore, it is important to call this method **before** executing the
  * query. Otherwise, the `Search` event will not be logged, and you will get a warning message in the console. See
- * [Sending Custom Analytics Events](https://developers.coveo.com/x/KoGfAQ).
+ * [Sending Custom Analytics Events](https://docs.coveo.com/en/365/).
```

Comment:
Link to https://docs.coveo.com/en/2726/#logging-your-own-search-events instead and update link name accordingly.